desafio-03/lucasfturos/cpp/MakeFile

```
@@ -0,0 +1,2 @@
+palindromo: palindromo.cpp
+	g++ -o -Wall palindromo.cpp palindromo
```

```suggestion
	g++ -o -Wall palindromo.cpp
```

Aq é uma lista de source files. `palindromo` não existe na primeira compilação:

```
: ~/projects/osprogramadores/op-desafios/desafio-03/lucasfturos/cpp
[0] % make
g++ -o -Wall palindromo.cpp palindromo
/usr/sbin/ld: cannot find palindromo: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [Makefile:2: palindromo] Error 1
```